### PR TITLE
Made new build additions to readme and Changed buildTest

### PR DESCRIPTION
**Pull request by Joperezc**
Changed buildTest to only show usable tests and allow coverage of individual tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,26 +41,38 @@
...
-    1. Run setup via command line, in the form (python setup.py (Mac/Linux)) (py setup.py (Windows))
+    1. Creating and entering a virtual environment
 
-    2. setup.py will create a viurutal environment, instatiate databases,
-       and intall any required non-base modules/packages into the environment
+        Activating
+            Linux/Mac
+                -use the command line command (python3 -m venv spell)
+            Windows
+                -Use the command line command (py -m venv spell)
```
Comment (nrhoopes) [inline README.md:50]: Parens '()' could be confusing, maybe use ' ?

File: README.md [hunk 2]
```diff
@@ -99,13 +111,27 @@
...
-    Command to run pytest (Be sure to be in environment before using)
-        Run All:
-            - (pytest UnitTests/)
-        Run Specific:
-            - (pytest UnitTests/TestFileName)
+    Using the Testing Suite
+
+    Running a complete Test
+        - Use the command line command ( (Test) or (Test all))
+            -To add coverage use ( (Test cov) or (Test all cov))
```
Comment (nrhoopes) [inline README.md:120]: Paren spacing looks bad ( (xyz))

Comment (Joperezc) [reply README.md]: Changed style to better emphasize commands

File: README.md (hunk 1 above)
Comment (Joperezc) [reply README.md]: Changed style to better emphasize commands

**Review by nrhoopes — APPROVED** [review]

**Pull request merged**